Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -227,6 +227,96 @@ extension SPNoteEditorViewController {
 }
 
 
+// MARK: - History
+//
+extension SPNoteEditorViewController {
+
+    /// Indicates if note history is shown on screen
+    ///
+    @objc
+    var isShowingHistory: Bool {
+        return historyViewController != nil
+    }
+
+    /// Shows note history
+    ///
+    @objc
+    func presentHistoryController() {
+        guard SPAppDelegate.shared().simperium.authenticator.connected else {
+            let title = NSLocalizedString("version-alert-message", comment: "Error alert message shown when trying to view history of a note without an internet connection")
+            let cancelTitle = NSLocalizedString("OK", comment: "")
+            let alertController = UIAlertController(title: title,
+                                                    message: nil,
+                                                    preferredStyle: .alert)
+            alertController.addCancelActionWithTitle(cancelTitle)
+            present(alertController, animated: true, completion: nil)
```

Comment:
Maybe move the AlertController snippet to another method, and decouple things? (OR was that gonna be embedded in the history?)

Reply:
I asked Sylvester about it and we're gonna remove this alert completely and will show an error message directly in the card. We will also adjust a copy to `Couldn't retrieve history`